src/SFA.DAS.ApplyService.Database/Tables/OrganisationSectorExperts.sql

```diff
@@ -0,0 +1,29 @@
+﻿CREATE TABLE [dbo].[OrganisationSectorExperts]
+(
+	[Id] INT NOT NULL IDENTITY(1,1),
+	[OrganisationSectorId] INT NOT NULL,
+	[FirstName] NVARCHAR(300) NOT NULL,
+	[LastName] NVARCHAR(300) NOT NULL,
+	[JobRole] NVARCHAR(300) NOT NULL,
+	[TimeInRole] NVARCHAR(300) NOT NULL,
+	[IsPartOfAnyOtherOrganisation] [BIT] NOT NULL,
+	[OtherOrganisationNames] NVARCHAR(1000) NULL,
+	[DateOfBirthMonth] TiNYINT NOT NULL,
+	[DateOfBirthYear] INT NOT NULL,
+	[ContactNumber] NVARCHAR(50) NOT NULL,
+	[Email] NVARCHAR(350) NOT NULL,
+	[SectorTrainingExperienceDuration] NVARCHAR(30) NULL,
+	[SectorTrainingExperienceDetails] NVARCHAR(MAX) NULL,
+	[IsQualifiedForSector] BIT NOT NULL,
+	[QualificationDetails] NVARCHAR(MAX) NULL,
+	[IsApprovedByAwardingBodies] BIT NOT NULL, 
+	[AwardingBodyNames] NVARCHAR(MAX) NULL, 
+	[HasSectorOrTradeBodyMembership] BIT NOT NULL, 
+	[SectorOrTradeBodyNames] NVARCHAR(MAX) NULL, 
+	[TypeOfApprenticeshipDelivered] NVARCHAR(50) NULL, 
+	[ExperienceInTrainingApprentices] NVARCHAR(100) NULL, 
```

Comment:
ExperienceInTrainingApprentices needs to be 255 characters

Reply:
Following are the expected input for this, so I guess 100 should be sufficient 
![image](https://user-images.githubusercontent.com/29398561/150170840-fdb7c4c8-1a89-4ad1-a7fd-9a1a34fd1c1a.png)